test/Table_Tests/src/Util.enso

```diff
@@ -5,32 +5,35 @@ import Standard.Database.DB_Column.DB_Column
 from Standard.Table import Column, Table
 
 from Standard.Test import all
+from Standard.Test.Extensions_Helpers import rhs_error_check
```

Comment:
I don't like this [friend API between Standard.Test and Table tests](http://wiki.apidesign.org/wiki/FriendDependencies). I'd rather copy the definition of `rhs_error_check` here.